op-service/client/dial.go

```diff
@@ -35,7 +33,7 @@ func DialEthClientWithTimeout(timeout time.Duration, log log.Logger, url string)
 
 // DialRollupClientWithTimeout attempts to dial the RPC provider using the provided URL.
 // If the dial doesn't complete within timeout seconds, this method will return an error.
-func DialRollupClientWithTimeout(timeout time.Duration, log log.Logger, url string) (*sources.RollupClient, error) {
+func DialRollupClientWithTimeout(timeout time.Duration, log log.Logger, url string) (*RollupClient, error) {
```

Comment:
I think we should move the dial function out of the `client` package, and keep `RollupClient` in the `sources` package.

Reply:
Moved to `op-service/dial`. Or do you have an alternate suggestion?